tests/garage/garage.spec.js

```diff
@@ -0,0 +1,24 @@
+import { test, expect } from '../../src/fixtures/customFixtures'
+import {CARS} from '../../src/data/cars'
+
+test.describe('Garage', ()=> {
+
+    let addCarPopup
+    let editFirstCarPopup
+    let removeCarPopup
```

Comment:
you use each of these components only in one place so moving declaration to describe is redundant